java/food-packaging/src/main/java/org/acme/foodpackaging/domain/PackagingSchedule.java

```diff
@@ -20,6 +20,10 @@ public class PackagingSchedule {
     @ProblemFactCollectionProperty
     private List<Product> products;
 
+    @ProblemFactCollectionProperty
+    @ValueRangeProvider
+    private List<Operator> operators;
```

Comment:
Operator is an entity! How come it doesn't fail when annotated as a fact?

Reply:
This test need to be updated to use a different class: https://github.com/TimefoldAI/timefold-solver/blob/204786bf00c046362c3539546364c00bedd34dcb/core/src/test/java/ai/timefold/solver/core/impl/domain/solution/descriptor/SolutionDescriptorTest.java#L135-L138

Reply:
https://github.com/TimefoldAI/timefold-solver/pull/1646